syntax = "proto3";

package calculator;

service Calculator {
    rpc AddNumbers(AddRequest) returns (AddResponse) {}
}

message AddRequest {
    int32 x = 1;
    int32 y = 2;
}

message AddResponse {
    int32 result = 1;
}
